<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="9979" started-at="2020-09-01T13:16:28 IST" finished-at="2020-09-01T13:16:38 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="9979" started-at="2020-09-01T13:16:28 IST" finished-at="2020-09-01T13:16:38 IST">
      <class name="work.assignment.Market_mutual_funds">
        <test-method status="PASS" signature="get_value()[pri:0, instance:work.assignment.Market_mutual_funds@6b9651f3]" name="get_value" duration-ms="6549" started-at="2020-09-01T13:16:28 IST" finished-at="2020-09-01T13:16:34 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- get_value -->
        <test-method status="FAIL" signature="mutual_funds()[pri:0, instance:work.assignment.Market_mutual_funds@6b9651f3]" name="mutual_funds" duration-ms="1696" started-at="2020-09-01T13:16:34 IST" finished-at="2020-09-01T13:16:36 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [<1463>] but found [1463]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [<1463>] but found [1463]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:617)
at work.assignment.Market_mutual_funds.mutual_funds(Market_mutual_funds.java:40)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mutual_funds -->
        <test-method status="PASS" signature="validateResponseJSONBody_test()[pri:0, instance:work.assignment.Market_mutual_funds@6b9651f3]" name="validateResponseJSONBody_test" duration-ms="1668" started-at="2020-09-01T13:16:36 IST" finished-at="2020-09-01T13:16:38 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateResponseJSONBody_test -->
      </class> <!-- work.assignment.Market_mutual_funds -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
